.github/workflows/release.yaml

```
@@ -130,7 +110,8 @@ jobs:
           github_token: ${{ secrets.GH_RUNNER_TOKEN }}
   deploy-staging:
     name: Deploy Gnosis Staging
-    runs-on: depot-ubuntu-22.04-2
+    if: github.event.inputs.binary == 'bloklid'
```

actionlint (1.7.12)

[error] property "binary" is not defined in object type {release_type: string}

(expression)